### PR TITLE
TSK: clue data

### PR DESCRIPTION
**Pull request by Chr1Z93**
contains the clue data for locations for these scenarios:
- 5-A
- 11-B
- 16-D
- 21-F
- 33-K

also contains the data for the difficulty selector

### CODE REVIEW

**Comment by Chr1Z93** [comment]
It should now include all the clue data for TSK (probably missing some non-obvious locations that are on the back of agenda/act - got one of these at least!)

File: src/core/DataHelper.ttslua [hunk 1]
```diff
@@ -723,6 +723,89 @@ LOCATIONS_DATA = JSON.decode(
...
+  "Rainy London Streets": {"type": "perPlayer", "value": 2, "clueSide": "back"},
+  "Big Ben": {"type": "perPlayer", "value": 1, "clueSide": "back"},
+  "Traitors' Gate": {"type": "perPlayer", "value": 1, "clueSide": "back"},
+  "The Tower Bridge": {"type": "perPlayer", "value": 1, "clueSide": "back"},
+  "Kensington Gardens": {"type": "perPlayer", "value": 1, "clueSide": "back"},
+  "Tower of London": {"type": "perPlayer", "value": 1, "clueSide": "back"},
+  "Westminster Abbey": {"type": "perPlayer", "value": 1, "clueSide": "back"},
+  "Tower Prison": {"type": "perPlayer", "value": 2, "clueSide": "back"},
+
+  "Saadian's Tombs_b9fc93": {"type": "perPlayer", "value": 3, "clueSide": "back"},
```
Comment (argonui) [inline src/core/DataHelper.ttslua:735]: Why do some have guids and others don't?

File: src/core/DataHelper.ttslua [hunk 2]
```diff
@@ -858,476 +941,704 @@ PLAYER_CARD_TOKEN_OFFSETS = {
 
 modeData = {
```
Comment (argonui) [inline src/core/DataHelper.ttslua:942]: Please have separate PRs. These two changes seem unrelated

**Review by argonui — CHANGES_REQUESTED** [review]

File: src/core/DataHelper.ttslua (hunk 1 above)
Comment (Chr1Z93) [reply src/core/DataHelper.ttslua:735]: GUIDs are only needed for locations that exist multiple times

File: src/core/DataHelper.ttslua (hunk 2 above)
Comment (Chr1Z93) [reply src/core/DataHelper.ttslua]: Once again the auto-formatting reformatted this...
Fixed it and now it should be clear why it's here 😅

**Review by BootleggerFinn — APPROVED** [review]

**Review by Buhallin — APPROVED** [review]

**Review by argonui — APPROVED** [review]
